testing/test_configs/integrations/azureboards_public.yaml

```diff
@@ -0,0 +1,2 @@
+origin: https://dev.azureboards.com/todoerruser/todocheck/
```

Comment:
This points to an unowned domain. Why is that?

Is this the correct URL, when I open it in my browser, I don't get anything.

Reply:
This was unintentional. Very good catch. I am wondering how the tests are passing with an invalid URL.. However I've updated both public and private URLs accordingly.